### PR TITLE
CTest setup

### PR DESCRIPTION
**Pull request by jensschmeink**
## Checklist:

- [ ] A test for the new functionality was added (if applicable).
- [ ] All tests run without failure.
- [ ] The changelog has been extended, if this MR contains important changes from the users's point of view.
- [ ] The new code complies with the GTlab's style guide.
- [ ] New interface methods / functions are exported via `EXPORT`. Non-interface functions are NOT exported.
- [ ] The number of code quality warnings is not increasing.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -64,6 +64,7 @@ windowsBuildDebug:
     - .build-win_20
     - .debugBuildRules
   script:
+    - $env:PATH+=(";"+$env:QT_INSTALL_PATH+"\bin;")
```
Comment (jensschmeink) [inline .gitlab-ci.yml:67]:
```suggestion
    - $env:PATH+=(";.\build;"+$env:GTLAB_DEV_TOOLS+"\binDebug;"+$env:GTLAB_DEV_TOOLS+"\binDebug\modules;")
```

**Comment by jensschmeink** [comment]
@rainman110 currently the pipeline is not triggered. May you check this? Maybe the problem with the token?

**Comment by rainman110** [comment]
@jensschmeink Strange. The code has been built, but the status seems not to be transferred:
https://gitlab.dlr.de/gtlab/internal/github-mirrors/gtlab-core-mirror/-/pipelines/776701

**Comment by jensschmeink** [comment]
@rainman110 as the pipeline seems to be passed please approve this change

**Review by rainman110 — APPROVED** [review]

**Pull request merged**